serial/serbridge.c

```diff
@@ -262,13 +707,54 @@ sendtxbuffer(serbridgeConnData *conn)
   return result;
 }
 
+
+//  Escape '\xFF' bytes in buffer for Telnet protocol before sending it over the
+//  air
+//
+static sint8 ICACHE_FLASH_ATTR
+espbuffsend_tn ( serbridgeConnData *conn, const char *data, uint16 len )
+{
+  //  How many bytes for the new buffer?
+  //
+  int n = 0 ;
+  for ( int i=0 ; i<len ; i++ )
+    if ( data[i] == 0xFF )
+      n++ ;
+
+  //  Allocate and populate new buffer if necessary
```

Comment:
have you thought of allocating a new buffer only for the extra characters? the esp8266 is somewhat memory constrained after all...